src/config/dynamic-theme-fixes.config

```diff
@@ -10388,7 +10377,7 @@ body
 
 ================================
 
-globo.com
+*.globo.com
```

Comment:
As you added asterisk in the site you need to move whole site fix up and put it after `*.crm*.dynamics.com` before `*.maricopa.edu`.

The sitefixes must be listed alphabetically.

Alternatively you can add `globo.com` before the line `*.globo.com` then it can stay in this position and both aliases will be used for sitefixes.

How you will fix it it's your choice. 😉